components/browse/BrowseChip.vue

```diff
@@ -1,30 +1,31 @@
 <template>
-  <b-badge
-    v-if="title"
-    :to="localePath({ name: 'entity-type-all', params: { type: type, pathMatch: path }})"
-    pill
-    variant="nocolor"
-    class="mb-3 mr-3 font-weight-normal bg-transparent"
-    data-qa="browse chip"
-  >
-    {{ title }}
-  </b-badge>
+  <li>
+    <b-badge
+      v-if="title"
+      :to="localePath({
```

Comment:
This is still coupled to the entity-type-all route.

The full path needs to be passed into the component so that here we need only:
```html
:to="linkTo"
```